### PR TITLE
Add dig_down_more option

### PR DESCRIPTION
**Pull request by SmithMatthew**
Please review this, @casey-collab

### CODE REVIEW

**Comment by SmithMatthew** [comment]
Please review this, @casey-collab

**Comment by pmallory** [comment]
@casey-collab please review this PR

File: english/you-are-in-minecraft/dig_down/dig_down.md [hunk 1]
```diff
@@ -24,4 +24,6 @@ Do you go [left?](goleft/goleft.md)
 
 Or maybe go [right?](goright/goright.md) 
 
+Do you continue to [dig down] (dig_down_more)
```
Comment (casey-collab) [inline english/you-are-in-minecraft/dig_down/dig_down.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/you-are-in-minecraft/dig_down/dig_down_more/dig_down_more.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+Against your better judgment, you continue to dig straight down.
```
Comment (casey-collab) [inline english/you-are-in-minecraft/dig_down/dig_down_more/dig_down_more.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by SmithMatthew** [comment]
@casey-collab Please review this. I fixed the incorrect file path.

**Comment by casey-collab** [comment]
Looks great! :icecream:

**Pull request merged**